package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Maybe this has been commited by mistake? Same question for the `countries.geo.json` file